features/raadpleeg-verblijfplaats-met-periode/gba/overzicht-gba.feature

```diff
@@ -0,0 +1,330 @@
+#language: nl
+
+@gba
+Functionaliteit: raadpleeg verblijfplaats voorkomens in een periode
+
+  Als consumer van de Historie bevragen API
+  wil ik kunnen opvragen waar een persoon in een periode heeft verbleven
+  zodat ik deze informatie kan gebruiken in mijn proces
+
+  Achtergrond:
+    Gegeven adres 'A1' heeft de volgende gegevens
+    | gemeentecode (92.10) | identificatiecode verblijfplaats (11.80) | straatnaam (11.10) |
+    | 0800                 | 0800010000000001                         | Laan               |
+    En adres 'A2' heeft de volgende gegevens
+    | gemeentecode (92.10) | identificatiecode verblijfplaats (11.80) | straatnaam (11.10) |
+    | 0800                 | 0800010000000002                         | Luttestraat        |
+    En adres 'A3' heeft de volgende gegevens
+    | gemeentecode (92.10) | identificatiecode verblijfplaats (11.80) |
+    | 0800                 | 0800010000000003                         |
+    En adres 'A4' heeft de volgende gegevens
+    | gemeentecode (92.10) | locatiebeschrijving (12.10) |
+    | 0800                 | Woonboot bij de Grote Sloot |
+
+Rule: Een verblijfplaats van een persoon met bekend datum aanvang adreshouding/adres buitenland wordt geleverd als datumTot van de gevraagde periode groter of gelijk is aan datum aanvang adreshouding/adres buitenland en in het geval van een volgende adreshouding/adres buitenland is de datumVan kleiner dan datum aanvang volgende adreshouding/adres buitenland
```

Comment:
ik denk voor datumTot niet "groter of gelijk", alleen groter. Bijvoorbeeld als aanvang=20200526 en gevraagde periode is van 2020-01-01 tot 2020-05-26 verwacht ik verblijfplaatsen tot en met 2020-05-25. De verblijfplaats die pas vanaf 2020-05-26 geldt moet dan niet geleverd worden.

Reply:
aangepast